prac_06/programming_language.py

```diff
@@ -0,0 +1,15 @@
+class ProgrammingLanguage:
+    def __init__(self, name="", typing="", reflection=True, year=0):
+        self.name = name
+        self.typing = typing
+        self.reflection = reflection
+        self.year = year
+
+    def __str__(self):
+        return f"{self.name}, {self.typing} Typing, Reflection={self.reflection}, First appeared in {self.year}"
+
+    def __repr__(self):
+        return f"{self.name}, {self.typing}, {self.reflection}, {self.year}"
+
+    def is_dynamic(self):
+        return self.reflection
```

Comment:
The 'is' in this class name indicates that we want to return a boolean. This function currently just returns either "Dynamic" or "Static". Consider using an if statement to check whether it's dynamic and return the result...